packages/map-template/src/components/WebComponentWrappers/Search/Search.jsx

```diff
@@ -21,8 +21,9 @@ import searchAllVenuesState from '../../../atoms/searchAllVenues';
  * @param {boolean} props.preventFocus - If set to true, the search field will be disabled.
  * @param {boolean} props.google - Set to true to include results from Google Places autocomplete service.
  * @param {boolean} props.mapbox - Set to true to include results from Mapbox Places autocomplete service.
+ * @param {number} amountOfSearchResults - The amount of search results to return.
```

Comment:
```suggestion
 * @param {number} props.amountOfSearchResults - The amount of search results to return.
```

The props object is destructured.